docs/content/preview/secure/authentication/ldap-authentication-ysql.md

```diff
@@ -39,15 +39,15 @@ These are described below.
 
 ## Simple Bind Mode
 
-In **simple-bind** mode, YB-TServer will bind to the Distinguished Name ("DN") constructed with "prefix username suffix" format. Here is an example for Simple bind mode:
+In **simple-bind** mode, YB-TServer binds to the Distinguished Name ("DN") constructed with "prefix username suffix" format. The following is an example using Simple bind mode:
```

Comment:
If simple-bind is an actual setting, it should be wrapped in code, not bold. (if it doesn't correspond to a UI element, bold shouldn't be used)